File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/strongtypesvb/vb/useraccess3.vb
' <snippetuseraccess3>



 ' Revoke access to the lead for the second user.
 Dim revokeUser2AccessReq = New RevokeAccessRequest With
                            {
                                .Revokee = systemUser2Ref,
                                .Target = leadReference
                            }

 Console.WriteLine("Revoking access to the lead for {0}..." &amp; vbCrLf,
                   GetEntityReferenceString(systemUser2Ref))
 _serviceProxy.Execute(revokeUser2AccessReq)

' </snippetuseraccess3>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/strongtypes/cs/useraccess3.cs
// <snippetuseraccess3>



 // Revoke access to the lead for the second user.
 var revokeUser2AccessReq = new RevokeAccessRequest
 {
     Revokee = systemUser2Ref,
     Target = leadReference
 };

 Console.WriteLine("Revoking access to the lead for {0}...\r\n",
     GetEntityReferenceString(systemUser2Ref));
 _serviceProxy.Execute(revokeUser2AccessReq);

// </snippetuseraccess3>